src/data/nat/prime.lean

```diff
@@ -593,6 +594,27 @@ lemma mem_factors {n p} (hn : 0 < n) : p ∈ factors n ↔ prime p ∧ p ∣ n :
 ⟨λ h, ⟨prime_of_mem_factors h, (mem_factors_iff_dvd hn $ prime_of_mem_factors h).mp h⟩,
  λ ⟨hprime, hdvd⟩, (mem_factors_iff_dvd hn hprime).mpr hdvd⟩
 
+lemma factors_subset {n k : ℕ} (h : k ≠ 0) : n.factors ⊆ (n * k).factors :=
```

Comment:
I think `0 < k` is more common as a hypothesis

Reply:
[The consensus on Zulip appears to have landed on `k ≠ 0` as a hypothesis.](https://leanprover.zulipchat.com/#narrow/stream/113489-new-members/topic/.60ne_zero.60.20vs.20.60pos.60.20for.20proofs.20involving.20.60nat.60)

Moreover, this statement should hold for rings in general, where there may not even be a `<` operator, so let's keep the `≠`.